### DISCUSSION TITLE
How to obtain multi-line logs from syslog

### DISCUSSION THREAD

**Discussion by xxsdnol** [vectordotdev/vector#18836] [Q&A]
How to obtain multi-line logs of type syslog in sources. It is currently known that multi-line logs of file and docker can be obtained. There is no problem in actual testing. If the input source is syslog, how should it be obtained?

The following is an example of how I can obtain a multi line log from a file. How can I modify it to obtain a multi line log from a syslog

[sources.multiline_logs]
type = "file"
include = [ "/opt/multiline.log" ]
	[sources.multiline_logs.multiline]
		start_pattern = '^<\d+>'
		mode = "halt_before"
		condition_pattern = '<\d+>'
		timeout_ms = 1000

**Comment by jszwedko** [comment]
Hi @XSWClevo !

You can use the `reduce` transform instead with any sources that don't have native `multiline` options.
